src/opnsense/mvc/app/views/OPNsense/OpenVPN/export.volt

```diff
@@ -180,8 +180,15 @@
                 $("#row_openvpn_export\\.p12_password").removeClass("ui_hide");
             }
         });
-    });
 
+        $("#accounts_search").on("keyup", function () {
```

Comment:
I wonder if `keyup` misses a right click paste, in which case `input` may also be an idea.